src/main/java/com/team701/buddymatcher/controllers/users/UserController.java

```diff
@@ -35,7 +32,7 @@ public UserController(UserService userService, ModelMapper modelMapper) {
 
     @ApiOperation("Get method to retrieve a user by id")
     @GetMapping(path = "/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
-    public ResponseEntity<UserDTO> one(@PathVariable("id") String id) {
+    public ResponseEntity<UserDTO> one(@PathVariable("id") Long id) {
```

Comment:
Nit: could this be changed to a more descriptive method name?

Reply:
Will do :)